argcomplete/_check_module.py

```diff
@@ -61,8 +62,25 @@ def find(name, return_package=False):
 
 
 def main():
-    with open(find(sys.argv[1])) as f:
-        head = f.read(1024)
+    try:
+        name = sys.argv[1]
+    except IndexError:
+        raise ArgcompleteMarkerNotFound('missing argument on the command line')
```

Comment:
This is only ever called [here](https://github.com/kislyuk/argcomplete/blob/b05ec4068830c9654bf5758bf5156ab78655df46/argcomplete/bash_completion.d/python-argcomplete.sh#L32) and it always passes an argument, even if that argument is the empty string.

I'm unfamiliar with Fedora's bug reporting; will it trigger even if the user types `python -m argcomplete._check_module` at the prompt? If so, this change is a reasonable protection against that. If not, I don't think it's necessary.

Reply:
https://bugzilla.redhat.com/show_bug.cgi?id=1737212 bug report was only about unhandled open() error. You can modify my PR to remove error checking on sys.argv if you prefer.

Reply:
On reflection, I'm okay with leaving this in. If this were using argparse, it would end up calling `sys.exit(1)` anyway, so this has roughly the same effect.